DependencyInjection/DoctrineExtension.php

```diff
@@ -343,6 +345,10 @@ protected function ormLoad(array $config, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('orm.xml');
 
+        if (class_exists(AbstractType::class) && method_exists(DoctrineType::class, 'reset')) {
```

Comment:
The second check is not needed as we require at least Symfony 3.4 which will always have the `reset` method. Since I'm not aware of Symfony 5 dropping it, we should not use the check.

```suggestion
        if (class_exists(AbstractType::class)) {
```

Reply:
I agree, removed extra check.